src/utils/textUtils.ts

```diff
@@ -10,3 +10,8 @@ export const formatAmount = (amount: number) => {
   parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',')
   return parts.join('.')
 }
+
+export const formatBlockNumber = (blockNumber: number | undefined) => {
+  if(!blockNumber) return ''
+  return blockNumber.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ')
```

Comment:
add comment on what it does please, not everyone are experts with regexp. Reasoning and example.
// blockNumber is a long string that we want to shorten
// ex. 123456789000987 -> 123...987